README.rst

```diff
@@ -29,8 +29,4 @@ From source:
 
 Documentation
 =============
-You can find the API doucmentation on the pytorch website: http://pytorch.org/docs/master/torchvision/
-
-Contributing
-============
-We appreciate all contributions. If you are planning to contribute back bug-fixes, please do so without any further discussion. If you plan to contribute new features, utility functions or extensions, please first open an issue and discuss the feature with us.
+You can find the API doucmentation on the pytorch website: http://pytorch.org/docs/master/torchvision/
```

Comment:
Can you revert this change and keep it as it was before?
Eventually, could you fix the typo in `doucmentation` -> `documentation`?

Reply:
Would you like me to preserve the entire README?